pages/[resourceType]/[id]/evaluate.tsx

```diff
@@ -226,11 +227,10 @@ const EvaluateMeasurePage = () => {
                       />
                     ) : (
                       <SelectComponent
-                        resourceType="Patient"
-                        setValue={setPatientValue}
-                        value={patientValue}
-                        disabled={true}
-                        placeholder="Patient selection disabled when 'Population' is selected"
+                        resourceType="Group"
+                        setValue={setGroupValue}
+                        value={groupValue}
+                        placeholder="Select a group or leave this field empty to run evaluate measure on entire population"
```

Comment:
This placeholder tasks get cut off depending on screen size (I didn't have the app in full screen and it cut off after "Select a group or leave this field empty to ...". This isn't that big of a deal so we don't need to fix it in this PR, but in the future we could consider adding a scroll bar to this so that users can see the full text on any screen size (pretty much all the other elements on the page are already responsive to screen size).

In the meantime, if you want to shorten the text a little, that might help - otherwise we can just leave it :)

Reply:
I ended up just using the standard placeholder text--I think the field and label text makes it pretty self explanatory